dashboard/src/components/AppView/DeploymentTable.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+
+import { IResource } from "shared/types";
+import DeploymentTable from "./DeploymentTable";
+import DeploymentItem from "./DeploymentItem";
+
+it("renders a deployment ready", () => {
+  const deployments = new Map<string, IResource>();
+  const dep = "foo";
+  deployments[dep] = {
```

Comment:
can we test with 2 deployments and ensure there are two DeploymentItem components rendered?